src/Frontend/Modules/Blog/Actions/Detail.php

```diff
@@ -208,6 +208,7 @@ private function parse()
             $this->record['meta_keywords'],
             ($this->record['meta_keywords_overwrite'] == 'Y')
         );
+		$this->header->setMetaCustom($this->record['meta_custom']);
```

Comment:
Seems like something gone wrong here with the indentation. Can you fix this @sweet-greg?